lib/factory.ex

```diff
@@ -1,6 +1,20 @@
 defmodule Community.Factory do
   use ExMachina.Ecto, repo: Community.Repo
 
+  def organization_factory do
+    %Community.Organization{
```

Comment:
Is there a benefit to this being in the database? Would it make sense to have this be a config option or a module that you fill in?

Reply:
The goal is to have it persist outside of the app config, so other communities can deploy (and not fork) the codebase and update the particulars about their community.  If it's in a config option, then they'd have to fork.

Reply:
Could you accomplish the same by recommending a file name `organization_config.exs` and using:

```elixir
import_config "organization_config.exs"
```

Reply:
they'd still have to fork.

Reply:
No:

```elixir

config :community, :organization, %Organization{name: "Triangle Design"}

if File.exists?("config/organization_config.exs") do
  import_config "organization_config.exs"
end
```

This allows us to have a default they fall back on, no Git changes required and no db calls.